fuel_test/check_version.sh

```diff
@@ -7,10 +7,10 @@
 [ $DEBUG == "on" ] && set -x
 
 if [[ ! -d "/tmp/fuel-plugin-xenserver" ]]; then
-	git clone https://review.openstack.org/openstack/fuel-plugin-xenserver /tmp/fuel-plugin-xenserver
+	git clone https://review.openstack.org/openstack/fuel-plugin-xenserver -b ${BRANCH} /tmp/fuel-plugin-xenserver
```

Comment:
I'm confused - why is/was this cloning from review.openstack.org? Surely it should be git.openstack.org?